firebase-firestore/src/main/java/com/google/firebase/firestore/index/IndexEntry.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore.index;
+
+/**
+ * Represents an index entry saved by the SDK in the local storage. Temporary placeholder, since
+ * we'll probably serialize the indexValue right away rather than store it.
```

Comment:
Please add "TODO(indexing)".

Note that we already committed `FieldIndex`. In https://github.com/firebase/firebase-android-sdk/pull/2963, FieldIndex also gets an `indexId`.